web_client/utils/analysis.jsx

```diff
@@ -141,14 +141,15 @@ export const runTask = (taskName, params, options = {}) => {
       .reduce(objectReduce, {})
   );
 
-  return (
-    rest({ path, type: 'POST', data })
-      .then(({
-        response: {
-          job: { _id: id }
-        }
-      }) => pollJob(id, title, taskName, pollInterval, 0, maxPolls))
-  );
+  return rest({ path, type: 'POST', data })
+    .then(({ response }) => response)
```

Comment:
This function already exists in underscore/lodash (it's called `_.property()`). But with the nice ES6 syntax, I don't see any reason not to just use `x => x.response`. It's more readable and is the new idiom for this exact operation.